lib/help/help.go

```diff
@@ -69,13 +68,24 @@ func Separator() string {
 // Returns Os dependent separator
 func Separators(os string) string {
 	switch os {
-	case "linux":
-		return string(filepath.Separator)
+	case "windows":
+		return "\\"
 	default:
-		return ""
+		fallthrough
+	case "unix":
+		return "/"
 	}
 }
 
+func AddPathSuffix(path, suffix, os string) string {
```

Comment:
__[golint]__
* [golint] exported function AddPathSuffix should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->